homeassistant/components/zoneminder.py

```diff
@@ -40,6 +41,29 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+class _Feature:
+    def __init__(self, version_str):
+        self._version = packaging.version.parse(version_str)
+
+    def is_supported(self):
+        """Return the remote instance support for this feature.
+
+        Returns False if this feature isn't supported on the currently
+        connected zoneminder instance. Otherwise returns True.
+        """
+        remote_version = ZM.get('version', None)
```

Comment:
We should move away from using globals. It's out of scope for this PR to fix that but please don't introduce new usage of the globals but instead have the `ZM` object be passed in.

Actually, it's kinda overkill to introduce a whole class for this. Why not just store `alarm_status_supported` in the ZM dict in the setup?